content/zh/docs/application-store/app-lifecycle-management.md

```diff
@@ -28,37 +28,37 @@ KubeSphere 集成了 [OpenPitrix](https://github.com/openpitrix/openpitrix)（
 
 1. 使用 `admin` 帐户登录 KubeSphere 控制台。点击左上角的**平台管理**，选择**访问控制**。转到**帐户角色**，点击**创建**。
 
-   ![创建角色](/images/docs/zh-cn/appstore/application-lifecycle-management/create-role.PNG)
+   ![创建角色](/images/docs/zh-cn/appstore/application-lifecycle-management/create-role.png)
```

Comment:
Check all the images in your local build to see if they are dead.

Reply:
All good.